applications/nrf5340_audio/src/audio/streamctrl.c

```diff
@@ -39,9 +39,20 @@ struct ble_iso_data {
 
 DATA_FIFO_DEFINE(ble_fifo_rx, CONFIG_BUF_BLE_RX_PACKET_NUM, WB_UP(sizeof(struct ble_iso_data)));
 
+ZBUS_SUBSCRIBER_DEFINE(button_sub, CONFIG_BUTTON_MSG_SUB_QUEUE_SIZE);
+ZBUS_SUBSCRIBER_DEFINE(le_audio_evt_sub, CONFIG_LE_AUDIO_MSG_SUB_QUEUE_SIZE);
+
 static struct k_thread audio_datapath_thread_data;
+static struct k_thread button_msg_sub_thread_data;
```

Comment:
```suggestion
static struct k_thread button_msg_subscr_thread_data;
```

Same for the other ones